lago/vm.py

```diff
@@ -226,13 +226,22 @@ def _libvirt_xml(self):
         with open(_path_to_xml('dom_template.xml')) as xml_fd:
             dom_raw_xml = xml_fd.read()
 
-        qemu_kvm_path = [
-            path
-            for path in [
-                '/usr/libexec/qemu-kvm',
-                '/usr/bin/qemu-kvm',
-            ] if os.path.exists(path)
-        ].pop()
+        capabilities_raw_xml = self.libvirt_con.getCapabilities()
+        capabilities_xml = lxml.etree.fromstring(capabilities_raw_xml)
+        qemu_kvm_path = capabilities_xml.findtext(
+            "guest[os_type='hvm']/arch[@name='x86_64']/domain[@type='kvm']"
+            "/emulator"
+        )
+
+        if not qemu_kvm_path:
+            # Nested virtualization not available
```

Comment:
If lago is executed inside a VM and if nested virtualization is disabled on the host then this query will return nothing, hence the comment.

But if hardware acceleration is disabled on the host, this query will return nothing too.

I propose to update the comment with: `hardware acceleration (or nested virtualization) not available`, what do you think ?